src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -171,6 +172,21 @@
         [password-input-panel :t/signing-phrase-description in-progress?])
       (when in-progress? [react/view styles/processing-view])]]))
 
+;; MAIN SEND TRANSACTION VIEW
+(defn- send-transaction-view [{:keys [scroll] :as opts}]
+  (let [amount-input (atom nil)]
+    (reagent/create-class
+     {:component-will-mount (fn [_]
+                              (.addListener react/keyboard
+                                            "keyboardDidShow"
```

Comment:
This is Android only. For iOS, there is separate event `keyboardWillShow` https://github.com/status-im/status-react/blob/develop/src/status_im/ios/core.cljs#L25